assets/blocks/quiz/index.js

```diff
@@ -5,5 +5,6 @@ import registerSenseiBlocks from '../register-sensei-blocks';
 
 import questionBlock from './question-block';
 import quizBlock from './quiz-block';
+import './quiz-store';
```

Comment:
We could standardize this file name for the different store files. We could just rename the `assets/blocks/course-outline/store.js` to `outline-store.js`, since we have other stores with custom names.

Reply:
Renamed this in 3153068